xl2times/datatypes.py

```diff
@@ -138,6 +138,22 @@ class TimesXlMap:
     filter_rows: Dict[str, str]
 
 
+class TimesModel:
```

Comment:
Perhaps we want to wrap this in a `@dataclass`, so that it automatically generates `__str__`, `__init__`, etc. It might also be a bit clearer to pass the empty sets to the constructor when we construct the model, to make it obvious that we start with empty regions.

https://stackoverflow.com/questions/47955263/what-are-data-classes-and-how-are-they-different-from-common-classes